.github/workflows/wheels-build.yml

```diff
@@ -173,20 +173,16 @@ jobs:
   wheels-linux-aarch64:
     name: "Wheels / Linux AArch64"
     if: (inputs.wheels-linux-aarch64 == 'default' && inputs.default-action || inputs.wheels-linux-aarch64) == 'build'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ inputs.python-version }}
       - uses: dtolnay/rust-toolchain@stable
-      - uses: docker/setup-qemu-action@v3
-        with:
-          platforms: all
       - uses: pypa/cibuildwheel@v2.22.0
         env:
           CIBW_ARCHS_LINUX: aarch64
```

Comment:
We probably can also remove this line since it's implicitly the host arch. We only needed this to enable the emulation when running from x86.

Reply:
The tests of Pythons 3.9 and 3.10 had passed, so I've pushed up cd85bf783d that removes this explicit selection.